changelog/1.5.0.md

```diff
@@ -2,8 +2,10 @@
 
 ## `testkit`
 
-  * We have introduced `testkit`, an infrastructure to test metaprograms like a pro.
-    See [https://github.com/scalameta/scalameta/commit/506c284fd389fcbd12c47550a8527a9d051aa3b4](https://github.com/scalameta/scalameta/commit/506c284fd389fcbd12c47550a8527a9d051aa3b4) for more information.
+  * `testkit` is a new module that can be used to run property-based tests
```

Comment:
Just a URL should be enough, so that we're sure that people reading the notes understand what's going on :)